src/main/java/com/vmware/vipclient/i18n/messages/api/opt/local/LocalMessagesOpt.java

```diff
@@ -43,6 +43,7 @@ public void getComponentMessages(MessageCacheItem cacheItem) {
 					getResource(path.toString()).toURI());
 			Map<String, String> messages = JSONBundleUtil.getMessages(path);
 	    	cacheItem.addCachedData(messages);
+	    	cacheItem.setTimestamp(System.currentTimeMillis());
```

Comment:
Suggest to set 'never expire'

Reply:
It has to expire becasue maybe the service was just temporarily down. So that when it expires, fetch from service can be re-tried.

Reply:
When service is empty, it should be set as never expire.